clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/handlers/LoadBalancerUpsertHandler.groovy

```diff
@@ -42,15 +42,17 @@ class LoadBalancerUpsertHandler {
                                   List<Listener> listeners, Collection<String> securityGroups) {
     def amazonErrors = []
     def loadBalancerName = loadBalancer.loadBalancerName
-
-    if (securityGroups) {
-      loadBalancing.applySecurityGroupsToLoadBalancer(new ApplySecurityGroupsToLoadBalancerRequest(
-        loadBalancerName: loadBalancerName,
-        securityGroups: securityGroups
-      ))
-      task.updateStatus BASE_PHASE, "Security groups updated on ${loadBalancerName}."
+    if (!securityGroups) {
```

Comment:
I think you want to add an additional check here for folks (mostly in OSS, but still to some extent at Netflix) editing load balancers in EC2-Classic, since they will not have security groups.
```java
if (!securityGroups && loadBalancer.getVPCId() != null) {
```